refurbish/src/Refurbish/Docker.hs

```diff
@@ -13,7 +13,7 @@ qemuRunnerName = "refurbish-qemu-runner"
 
 dockerFile :: String
 dockerFile =
-  unlines [ "FROM ubuntu:18.04"
+  unlines [ "FROM ubuntu:20.04"
```

Comment:
Just FYI, this was necessary for some of the tests to pass (we needed a newer version of QEMU).

Reply:
Oh boy, that sounds exciting.  Do you happen to know the required version of qemu? It would be helpful to have that recorded in maybe the README

Reply:
I just tried with the version I compiled from `master`, then switched this version and it noticed that it works with this package too... The version in 20.04 is 4.2-3ubuntu6.1, and in 18.04 it's 2.11+dfsg-1ubuntu7.26. I could try to bisect, or we could leave it at that?

Reply:
Known to work on 4.2 is reasonable, no need to bisect

Reply:
I'll add a line in the README.